examples/sonoff_basic_toggle/toggle.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+typedef enum {
+    toggle_event,
+} toggle_event_t;
+
+typedef void (*toggle_callback_fn)(uint8_t gpio_num, toggle_event_t event);
```

Comment:
I do not think you need second argument here at all (and thus the enum).

Reply:
So delete lines 3-5 and remove ", toggle_event_t event" ?

Reply:
Yes. And simplify code in main.c accordingly.